torch/csrc/jit/passes/fold_conv_bn.cpp

```diff
@@ -87,15 +156,21 @@ void addBiasForConvIfNone(Module& module, const std::string& pattern_name) {
 
   if (is_floating_point_conv) {
     if (!t->hasAttribute("bias")) {
-      auto optional_tensor_type = OptionalType::create(TensorType::get());
-      t->addAttribute("bias", optional_tensor_type, true);
-      auto optional_tensor = c10::optional<at::Tensor>();
-      module.setattr("bias", optional_tensor);
+      t->addAttribute("bias", TensorType::get(), true);
+      module.setattr("bias", at::Tensor());
       replaceConvBiasWithGetAttr(module);
+    } else {
+      if (!module.attr("bias").isTensor()) {
+        // t->unsafeChangeAttributeType("bias", TensorType::get());
+        t->unsafeRemoveAttribute("bias");
+        t->addAttribute("bias", TensorType::get(), true);
```

Comment:
Not sure if this is the right way to change attribute type.
`unsafeChangeAttributeType` doesn't work here, since it won't allow parameter to be changed.